client/src/components/common/footer/style.css

```diff
@@ -0,0 +1,53 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway&display=swap");
+.footer {
+  background-color: #eff6f5;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.container {
+  display: flex;
+  font-size: 16px;
+  text-align: left;
+  padding: 4em 8vw 2em 6vw;
+}
+
+.footer__p1 {
+  font-size: 1em;
+  font-family: "Raleway", sans-serif;
+  color: #000000;
+}
+
+.footer__p2 {
+  width: min(37em, calc(100% - 2vw));
+  font-size: 1em;
+  font-family: "Raleway", sans-serif;
+  color: #000000;
+}
+
+.footer__lists {
+  display: flex;
+  text-align: left;
+  font-size: 1em;
+  font-family: "Raleway", sans-serif;
+  line-height: 28px;
+  padding: 3em 6vw 2em 1vw;
+}
+
+ul {
+  list-style-type: none;
+}
+
+@media screen and (max-width: 600px) {
+  .container {
+    display: flex;
+    flex-direction: column;
+    font-size: 12px;
+  }
+
+  .footer__lists {
+    font-size: 1em;
+    padding: 1em 6vw 1em 0vw;
```

Comment:
* always use fixed values for padding and margins (no percentage, no vw...etc.)
* Don't mix units. Either use `em` all the way or `vw` or `px` or whatever. Preferably either `px, rem or em`.